tripper/namespace.py

```diff
@@ -197,6 +258,10 @@ def __eq__(self, other):
 TIME = Namespace("http://www.w3.org/2006/time#")
 FNO = Namespace("https://w3id.org/function/ontology#")
 
+# EMMO = Namespace("http://w3id.org/emmo#")
+# MAP = Namespace("http://w3id.org/domain-mappings#")
+# DM = Namespace("http://w3id.org/datamodel#")
+# OTEIO = Namespace("http://w3id.org/oteio#")
```

Comment:
When should these be updated? Why not already?

Reply:
It is definitely something we should do, but I think it is better to do it in a separate PR, since it may have some implications on existing tests in other repositories.

It is probably a good idea to make a release before and after changing these.

We must also rename the datamodel and oteio repos to domain-datamodel and domain-oteio for the w3id redirections to work.